dpctl/_sycl_device.pyx

```diff
@@ -2045,6 +2043,18 @@ cdef class SyclDevice(_SyclDevice):
                 else:
                     return str(relId)
 
+    def get_device_id(self):
```

Comment:
Add a docstring

Reply:
This function requires `self` to an unpartitioned device. It is useful to add function `SyclDevice.get_unpartitioned_parent_device` to return the unpartitioned parent device that this device was obtained from.